documentation/writing_plans.md

```diff
@@ -391,6 +391,72 @@ plan loop(TargetSpec $targets) {
 
 If your plan accepts a single `TargetSpec` parameter you can call that parameter `targets` so that it can be specified with the `--targets` flag from the command line.
 
+### Creating Target objects
+
+There are two main ways you may want to instantiate Target objects within a plan: as a barestring
```

Comment:
It might be helpful to explain what creating a Target _means_. This section currently covers a bit of both a) adding new data to the inventory, and b) getting access to a Target object for data in the inventory, separate from whether or not it was already present in the inventory.

Reply:
Updated